sw/airborne/firmwares/rotorcraft/guidance/guidance_indi_hybrid.c

```diff
@@ -637,7 +694,7 @@ struct FloatVect3 nav_get_speed_sp_from_line(struct FloatVect2 line_v_enu, struc
   struct FloatVect2 final_vector;
   VECT2_SMUL(final_vector, direction, desired_speed/length_direction);
 
-  struct FloatVect3 speed_sp_return = {final_vector.x, final_vector.y, gih_params.pos_gainz*(ned_target.z - stateGetPositionNed_f()->z)};
+  struct FloatVect3 speed_sp_return = {final_vector.x, final_vector.y, (gih_params.pos_gainz)*(ned_target.z - stateGetPositionNed_f()->z)};
```

Comment:
not needed?

Reply:
yes you are right will change it now, it is a left over of the scheduling structure